### DISCUSSION TITLE
pfps, 第四。渠道。

### DISCUSSION THREAD

**Discussion by TheProdigyLeague** [community/community#51070] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

What does the github community think about [第四頻道。點。組織。]
brief rundown, 
An "anonymous" message board.
And before you call me a troll,
**Note**: Trolling as defined by general consensus,
_"Any citizen who posts online with **ill intent** by way of v-chat, I.E. inflammatory, insincere, digressive, extraneous, off-topic, pm's hate comments, or IRL, with intent of provoking others, manipulating data, etc.)"_
And, nobody is really anonymous, anyways. 
Avancer...
I am being sincere, as it has not been discussed here. Therefore, I am starting to think it has been condemned as archaic archival research or that I will be retaliated against.
Either way,
The admins of the channel will literally install back-doors if you simply visit the site. (I have seen unwanted notifications flash on my devices, and my devices usually go malfunct afterward.) 
So, what do you think about it? 
I have seen many a discussion but none here so far...
This stigma about the whole Guy Fawkes masks pfps, I mean, he's just a DC Comic fictional hero after all. 
But, how does Github general feel about it? 
That whole stigma of "you must be script kiddie, you must be fourteen year old just discovering loic" is in and of itself proselytizing. 
We're talking about cyber-criminals and trolls making the internet worse not vidya mods and vac bans here.
In short, please help me to understand where this comes from. I see it as internet history waiting to be studied by Sanctuary and our future civilization. Not some gate-keepy retort of "You're not apart of the cool kids club."
